src/js/view/view.js

```diff
@@ -893,9 +932,9 @@ function View(_api, _model) {
         return _model.get('isFloating') ? _wrapperElement : _playerElement;
     }
 
-    function _updateFloating(intersectionRatio) {
+    function _updateFloating(intersectionRatio, forceFloat) {
```

Comment:
again not in love with this, will give it some thought but happy to get suggestions